dss/api/files.py

```diff
@@ -139,6 +152,16 @@ def get_helper(uuid: str, replica: Replica, version: str = None, token: str = No
     return response
 
 
+def determine_filename(bundle_fqid, file_fqid):
+    """
+    Returns the filename for a file from its associated bundle metadata or None if not found in the bundle metadata.
+    """
+    if bundle_fqid:
```

Comment:
I'd pull this conditional up. See above comment

Reply:
Can do.